.github/workflows/build.yml

```diff
@@ -16,6 +16,15 @@ jobs:
       run: |
         sudo apt-get update
         sudo apt-get install -y libssl-dev libcurl4-openssl-dev uncrustify libyaml-dev
+    - name: Prepare libwebsockets
+      run: |
+        git clone https://libwebsockets.org/repo/libwebsockets
```

Comment:
We should clone at a specific tag or commit rather than `HEAD`

Reply:
Done. 
I use the latest stable branch `v4.2-stable`